samples/camera/camera2/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,7 @@
 
     <uses-permission android:name="android.permission.CAMERA" />
     <uses-permission android:name="android.permission.RECORD_AUDIO" />
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
```

Comment:
I don't see it being used btw. Is it needed?

Reply:
I originally meant for it to be used to send a quick notification so the user can view the photos in an external viewer, but not needed anymore